sphinx/builders/html.py

```diff
@@ -136,7 +167,7 @@ class StandaloneHTMLBuilder(Builder):
     script_files = ['_static/jquery.js', '_static/underscore.js',
                     '_static/doctools.js']  # type: List[unicode]
     # Ditto for this one (Sphinx.add_stylesheet).
-    css_files = []  # type: List[Dict[unicode, unicode]]
```

Comment:
\<off-topic>
`builder.css_files = [...]` by user is bypass this CSSContainer protection. If CSSContainer is implemented as descriptor, such breaking access will be avoided.
However, I think such descriptor implementation is not needed until such a problem is reported.
\</off-topic>